ci-operator/step-registry/ipi/aws/pre/ipi-aws-pre-chain.yaml

```diff
@@ -3,6 +3,12 @@ chain:
   steps:
   - chain: ipi-conf-aws
   - chain: ipi-install
+  env:
+    - name: COMPUTE_WORKER
+      default: 'null'
+      documentation: |-
+        The env var COMPUTE_WORKER defines type of flavour used for worker nodes, e.g. on AWS some tests may require larger m4.xlarge machines.
```

Comment:
nit: "The env var COMPUTE_WORKER" is already clear from context.  Maybe just open with "The instance type to use for compute nodes, ..." or some such?  Maybe link https://aws.amazon.com/ec2/instance-types/ ?

Personally, I'd prefer platform-agnostic values like `memory_24G_cpu_10` that got translated in the step to a suitable instance type.  That would allow jobs that needed larger compute to be platform-agnostic without needing folks to guess "hmm, I bet they don't need _all_ of the m4.xlarge properties, and I can use $WHATEVER when I pivot the job to GCP", etc.  But I'm fine with platform-specific values as a first pass.